cryptoconditions/condition.py

```diff
@@ -160,11 +216,20 @@ def hash(self, value):
 
         Args:
             value (Buffer): Hash as binary.
+
+        Raises:
+            Exception If hash value is not 32 bytes (256 bits) long.
         """
+        # TODO Use more precise Exception class
+        if len(value) != 32:
+            raise Exception(
```

Comment:
addressed in https://github.com/bigchaindb/cryptoconditions/pull/67/commits/91d6852149c6712a92c589b1ea38f8d50d7537d6